recipes/scikit-build-core/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set name = "scikit-build-core" %}
+{% set version = "0.1.0b2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/scikit_build_core-{{ version }}.tar.gz
+  sha256: 3d6bcc8ec84972a928962dd8894b82e24390f461a26fede87b4678dd1e98a8d6
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - hatch-vcs
+    - hatchling
+    - pip
+    - python >=3.7
+  run:
+    - python >=3.7
+    - cmake
```

Comment:
This and some others are not listed upstream. As you're both upstream and maintainer and familiar with conda-forge, I assume that they are needed?

Reply:
They are added by the `get_requires_for_build_wheel` (and sdist) hooks (and some of them are also in the pyproject extra, minus cmake & ninja) The one extra requirement is `make` - sadely, Conda-build injects `CMAKE_GENERATOR="Unix Makefiles"` even if make is not installed (and Ninja is)!

Reply:
Ahh ok makes sense.